.github/workflows/demo_deploy_ui.yml

```diff
@@ -43,7 +43,7 @@ jobs:
         env:
           IMAGE_TAG: ${{ steps.version.outputs.VERSION }}
         run: |
-          docker build -t opid/issuernode_ui:${{ env.IMAGE_TAG }} .
+          docker build -t wakeup-labs/issuernode_ui:${{ env.IMAGE_TAG }} .
```

Comment:
isn't it `wakeuplabs` like without the `-`?

Reply:
Woops, I missed it, I was sure I had replaced it, ty!